modules/apps/commerce/commerce-site-initializer/commerce-site-initializer-extender/src/main/java/com/liferay/commerce/site/initializer/extender/CommerceSiteInitializerImpl.java

```diff
@@ -730,6 +731,16 @@ private void _addOrUpdateCommerceCatalogs(
 					resourcePath, ".json", ".specification.options.json"),
 				serviceContext, servletContext);
 
+			String assetVocabularyName = jsonObject.getString(
+				"assetVocabularyName");
+
+			if (Validator.isNull(assetVocabularyName)) {
+				assetVocabularyName = siteName;
+			}
```

Comment:
I think this can be simplified with the `GetterUtil.getString(jsonObject.getString(
				"assetVocabularyName"), siteName)`

Reply:
empty string check is not accounted for, keeping